metered/src/metric.rs

```diff
@@ -63,6 +63,12 @@ pub trait Counter: Default + Clear + Clearable + Serialize {
     fn incr(&self);
 }
 
+/// A batch trait for Counters
+pub trait BatchCounter<T>: Default + Clear + Clearable + Serialize {
```

Comment:
Could you make the trait bound `Counter`

Reply:
done